dashboard/cron.py

```diff
@@ -639,12 +639,14 @@ def do(self):
             logger.info("** informational")
             status += self.update_unizin_metadata()
 
-        courses_added_during_cron: List[int] = list(
-            set(Course.objects.get_supported_courses().values_list('id', flat=True)) - set(self.valid_locked_course_ids))
+        all_str_course_ids = set(
+            str(x) for x in Course.objects.get_supported_courses().values_list('id', flat=True)
+        )
+        courses_added_during_cron: List[str] = list(all_str_course_ids - set(self.valid_locked_course_ids))
         if courses_added_during_cron:
-            logger.warning(
+            logger.debug(
                 f'During the run, users added {len(courses_added_during_cron)} course(s): {courses_added_during_cron}')
-            logger.warning(f'No data was pulled for these courses.')
+            logger.debug(f'No data was pulled for these courses.')
```

Comment:
@jonespm, I kept this in, just downgraded to `debug`. Let me know if you have an objection.

Reply:
Yeah that seems fine either as a debug or info, makes sense downgrading.